CHANGELOG.md

```diff
@@ -124,6 +127,8 @@ for further information about branching and tagging conventions.
 - Fixed bug with getting asset pair price in opportunity creation form. Now
   opportunity creation should work for both inverted and straight asset pairs.
   (Reit)
+- Getting sales from v3/sales
```

Comment:
* "Now using /v3 API for getting sales"
* and it's changed, not fixed

Reply:
ok